Lib/ufo2ft/outlineCompiler.py

```diff
@@ -1489,13 +1489,19 @@ def setupTable_post(self):
         post.glyphOrder = self.glyphOrder
 
     def setupOtherTables(self):
+        self.instructionCompiler = InstructionCompiler(
+            self.ufo, self.otf, autoUseMyMetrics=self.autoUseMyMetrics
+        )
+
         self.setupTable_glyf()
+
         if self.ufo.info.openTypeGaspRangeRecords:
             self.setupTable_gasp()
```

Comment:
Conceptually the gasp table also belongs to TrueType instructions, should it be moved to the `InstructionCompiler`?

Reply:
it does. yes please do move it, thanks